votor/src/consensus_pool/vote_certificate_builder.rs

```diff
@@ -48,36 +46,10 @@ pub struct VoteCertificateBuilder {
     input_bitmap_2: BitVec<u8, Lsb0>,
 }
 
-impl TryFrom<CertificateMessage> for VoteCertificateBuilder {
```

Comment:
Hmm, I remember why we want this now.

The theory is when you are doing rewards, you might want to incorporate other people's certificates into your cert, so you get even more votes in addition to the ones you already have.

Do we not need that any more?

Reply:
Can we wait maybe a little bit until the rewards are implemented? Put a TODO here to remove after we implement rewards and we are definitely sure we don't need this.

Or do you have rewards SIMD written which says rewards will only be given from votes received, not from certificates received? Let's make a decision on this.